examples/notebooks/stationarity_detrending_0.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stationarity in time-series\n",
+    "\n",
+    "Stationarity means that the statistical properties of a a time series i.e. mean, variance and covariance do not change over time. Most statistical models require the series to be stationary to make effective and precise predictions.\n",
```

Comment:
Maybe "Many statistical models" rather than "Most statistical models".

Reply:
ok, I shall correct it.